app/src/main/java/com/eveningoutpost/dexdrip/NSClientReceiver.java

```diff
@@ -199,6 +199,8 @@ private String toBgReadingJSON(HashMap<String, Object> sgv_map) {
         JSONObject jsonObject = new JSONObject();
         try {
             jsonObject.put("uuid", UUID.randomUUID().toString());
+            //TODO: Determine impact of making this not-null - scan all code
```

Comment:
I do not fully understand this TODO comment. Please can you explain it and maybe solve it, too?